src/app/widgets/Rotary/index.styl

```diff
@@ -25,6 +25,10 @@
     overflow: auto;
 }
 
+.wiringMessage{
+    padding: 2rem;
+}
+
```

Comment:
Best practice for naming classes is to use kebab case so we can easily know that they are in fact css classes. Ex. `.wiring-message {`